### PR TITLE
Fixes #596.

### LINKED ISSUE #596: JSON Message Body is not displayed correctly, if the message contains HTML
The JSON formatting gets messed up, when the message contains HTML tags:

untitled

### PR DESCRIPTION
**Pull request by HEskandari**
@Particular/serviceinsight-maintainers Please review. This should include @adamralph's comments on previous PR.

### CODE REVIEW

File: src/ServiceInsight/ServiceControl/DefaultServiceControl.cs [hunk 1]
```diff
@@ -6,25 +6,27 @@
...
     using RestSharp.Deserializers;
+    using Saga;
     using Serilog;
-    using ServiceInsight.Framework.Events;
-    using ServiceInsight.Framework.MessageDecoders;
-    using ServiceInsight.Framework.Settings;
-    using ServiceInsight.Models;
-    using ServiceInsight.Saga;
-    using ServiceInsight.Settings;
+    using Settings;
```
Comment (adamralph) [inline src/ServiceInsight/ServiceControl/DefaultServiceControl.cs]: These using statement changes seem unrelated to the fix. Can they be reverted?

File: src/ServiceInsight/ServiceControl/DefaultServiceControl.cs [hunk 2]
```diff
@@ -521,6 +521,8 @@ void LogError(IRestResponse response)
 
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
+        static string CleanUp(string content) => content.StartsWith(byteOrderMarkUtf8) ? content.Remove(0, byteOrderMarkUtf8.Length) : content;
```
Comment (adamralph) [inline src/ServiceInsight/ServiceControl/DefaultServiceControl.cs]: Could we replace the method on line 481? And keep the same method name?

**Comment by HEskandari** [comment]
It should be quite clear what has changed now: https://github.com/Particular/ServiceInsight/compare/cleanup-unicode-header

**Comment by distantcam** [comment]
This looks good now. 👍

**Comment by HEskandari** [comment]
@adamralph should be good now. 🤞
